langchain4j-vertex-ai-gemini/src/main/java/dev/langchain4j/model/vertexai/gemini/VertexAiGeminiChatModel.java

```diff
@@ -311,21 +311,12 @@ public VertexAiGeminiChatModel(
                 .setCustomHeaders(Collections.singletonMap("user-agent", "LangChain4j"))
                 .build();
 
-        this.generativeModel = new GenerativeModel(modelName, vertexAI)
-                .withGenerationConfig(generationConfig);
+        this.generativeModel = new GenerativeModel(modelName, vertexAI).withGenerationConfig(generationConfig);
 
         this.maxRetries = getOrDefault(maxRetries, 2);
 
-        if (logRequests != null) {
-            this.logRequests = logRequests;
-        } else {
-            this.logRequests = false;
-        }
-        if (logResponses != null) {
-            this.logResponses = logResponses;
-        } else {
-            this.logResponses = false;
-        }
+        this.logRequests = Objects.requireNonNullElse(logRequests, false);
```

Comment:
```suggestion
        this.logRequests = getOrDefault(logRequests, false);
```
To keep it consistent